### PR TITLE
  CPOL-101 Change default messaging for some table entries.

### PR DESCRIPTION
**Pull request by spinder**
Couldn't get spacing right.

### CODE REVIEW

File: src/properties/Messages.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@
 //Capture some strings we reuse. Possibly use in i18n later?
 
 export class Messages {
+    public static readonly tableDoubleDash: string = '--';
+    public static readonly tableNeverRun: string = 'Never run' ;
```
Comment (josejulio) [inline src/properties/Messages.ts]:
```suggestion
    public static readonly tableNeverRun: string = 'Never' ;
```

File: src/components/Policy/Table/PolicyTable.tsx [hunk 2]
```diff
@@ -92,8 +93,8 @@ const policiesToRows = (policies: PolicyRow[] | undefined): IRow[] => {
                     <>
                         <ExpandedContent
                             key={ policy.id + '-content' }
-                            description={ policy.description }
-                            conditions={ policy.conditions }
+                            description={ policy.description ? policy.description : Messages.tableDoubleDash + ' ' + Messages.tableNoDescription }
```
Comment (josejulio) [inline src/components/Policy/Table/PolicyTable.tsx]:
```suggestion
                            description={ policy.description ? Messages.tableNoDescription }
```

File: src/properties/Messages.ts [hunk 3]
```diff
@@ -2,6 +2,12 @@
...
+    public static readonly tableNoActions: string = 'No actions' ;
+    public static readonly tableNoConditions: string = 'No conditions' ;
+    public static readonly tableNoDescription: string = 'No description' ;
```
[the remark this answers is not in the document]
Comment (spinder) [reply src/properties/Messages.ts]: Making the changes.

**Comment by spinder** [comment]
I have made the requested changes.

**Pull request merged**